explore/src/main/scala/explore/users/UserPreferencesPopup.scala

```diff
@@ -185,91 +187,122 @@ object UserPreferencesContent:
     )
     .useStateView(RoleType.Pi)
     .useState(IsCleaningTheCache(false))
-    .render { (props, ctx, active, user, newKey, _, _, table, newRoleType, isCleaningTheCache) =>
-      import ctx.given
+    .useStateView(IsDeletingLayouts(false))
+    .render {
+      (
+        props,
+        ctx,
+        active,
+        user,
+        newKey,
+        _,
+        _,
+        table,
+        newRoleType,
+        isCleaningTheCache,
+        isDeletingLayouts
+      ) =>
+        import ctx.given
 
-      user.renderPot { ssoUser =>
-        val id   = ssoUser.user.id
-        val name = s"${ssoUser.user.givenName.orEmpty} ${ssoUser.user.familyName.orEmpty}"
-        val role = ssoUser.role.`type`.shortName
+        user.renderPot { ssoUser =>
+          val id   = ssoUser.user.id
+          val name = s"${ssoUser.user.givenName.orEmpty} ${ssoUser.user.familyName.orEmpty}"
+          val role = ssoUser.role.`type`.shortName
 
-        val allRoles = ssoUser.user.roles
+          val allRoles = ssoUser.user.roles
 
-        val requestCacheClean =
-          for {
-            _ <- isCleaningTheCache.setState(IsCleaningTheCache(true)).toAsync
-            _ <- ctx.workerClients.clearAll(
-                   isCleaningTheCache.setState(IsCleaningTheCache(false)).toAsync
-                 )
-          } yield ()
+          val requestCacheClean =
+            for {
+              _ <- isCleaningTheCache.setState(IsCleaningTheCache(true)).toAsync
+              _ <- ctx.workerClients.clearAll(
+                     isCleaningTheCache.setState(IsCleaningTheCache(false)).toAsync
+                   )
+            } yield ()
 
-        val cleanCacheButton =
-          Button(
-            label = "Clean local cache",
-            disabled = isCleaningTheCache.value.value,
-            icon = Icons.Trash,
-            onClick = requestCacheClean.runAsyncAndForget
-          ).small.compact
+          val cleanCacheButton =
+            Button(
+              label = "Clean local cache",
+              disabled = isCleaningTheCache.value.value,
+              loading = isCleaningTheCache.value.value,
+              icon = Icons.Trash,
+              onClick = requestCacheClean.runAsyncAndForget
+            ).small.compact
 
-        val unsupportedRoles = Enumerated[RoleType].all.filterNot { rt =>
-          allRoles.exists(r => r.`type` === rt)
-        }
-        val currentKeyRoleId = allRoles.find(_.`type` === newRoleType.get).map(_.id)
+          val deleteGridLayoutsButton =
```

Comment:
I wonder if it shoud be dev only. OTOH I can imagine more people wanting to reset their layouts